src/components/AddressSearch/index.tsx

```diff
@@ -291,36 +174,36 @@ function AddressSearch({
         if (!values.street && details.adr_address) {
             const streetAddressRegex = /<span class="street-address">([^<]*)<\/span>/;
             const adr_address = details.adr_address.match(streetAddressRegex);
-            const streetAddressFallback = lodashGet(adr_address, [1], null);
+            const streetAddressFallback = adr_address ? adr_address[1] : null;
```

Comment:
Either make sur eit's not needed or add ?. here:
```suggestion
            const streetAddressFallback = adr_address ? adr_address?.[1] : null;
```